client/src/commands/CleanCommand.ts

```diff
@@ -0,0 +1,15 @@
+import HardhatTaskCommand from "./HardhatTaskCommand";
+
+export default class CleanCommand extends HardhatTaskCommand {
```

Comment:
I think it's possible, but I thought we were going to focus more on commands and have tasks just for completion sake. The reason for this was that commands allow us much more flexibility. Also since I already had an implementation for the flatten command, which couldn't be ran as a task (at least I couldn't find how), adding more commands that invoke hardhat was just trivial. But I can make it invoke the task without problem. 

A small detail, instead of writing to the output channel, tasks open a new terminal window to show the output. I didn't find a way to open that exact kind of terminal for commands output. So the result is that tasks and commands have different UI for outputs. By having a command run a task, it could be a bit inconsistent with the other commands' output.